src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2405,33 +2406,60 @@ private static byte[] ConvertEnvVarsToByteArray(StringDictionary sd)
 
         private void SetStartupInfo(ProcessStartInfo startinfo, ref ProcessNativeMethods.STARTUPINFO lpStartupInfo, ref int creationFlags)
         {
-            bool hasRedirection = false;
+            // If we are starting a process using the current console window, we need to set its standard handlers
+            // explicitly when they are not redirected because otherwise they won't be set and the new process will
+            // fail with the "invalid handler" error.
```

Comment:
```suggestion
            // If we are starting a process using the current console window, we need to set its standard handles
            // explicitly when they are not redirected because otherwise they won't be set and the new process will
            // fail with the "invalid handle" error.
```

Reply:
Oops, there are also 2 other "handler" in the comment below

Reply:
Ah, sorry for the typos. Will fix soon.